tests/istio.mk

```diff
@@ -79,7 +79,7 @@ e2e_mixer: istioctl generate_yaml
 	go test -v -timeout 20m ./tests/e2e/tests/mixer -args ${E2E_ARGS} ${EXTRA_E2E_ARGS}
 
 e2e_bookinfo: istioctl generate_yaml
-	go test -v -timeout 20m ./tests/e2e/tests/bookinfo -args ${E2E_ARGS} ${EXTRA_E2E_ARGS}
+	go test -v -timeout 60m ./tests/e2e/tests/bookinfo -args ${E2E_ARGS} ${EXTRA_E2E_ARGS}
```

Comment:
Does it take 60 min to run bookinfo ? 

Can we move only the vm test to separate target if it's that long, so it runs in parallel ?